server/server.go

```diff
@@ -7,7 +7,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	_ "net/http/pprof"
+	_ "net/http/pprof" //nolint:gosec
```

Comment:
This one is tricky. The warning is about the debug endpoint that is exposed. But we have a flag that enables this so I think it's safe.